app/javascript/dashboard/components/widgets/conversation/ConversationCard.vue

```diff
@@ -111,6 +112,26 @@ export default {
       return this.chat.meta;
     },
 
+    thumbnailBadge() {
+      if (this.isATwilioChannel) {
+        if (this.isATwilioSMSChannel) {
+          return 'sms';
+        }
+        if (this.isATwilioWhatsappChannel) {
+          return 'whatsapp';
+        }
+      }
+      if (this.isATwitterInbox) {
+        if (this.chat.additional_attributes.type === 'tweet') {
+          return 'twitter-tweet';
+        }
+        if (this.chat.additional_attributes.type === 'message') {
+          return 'twitter-message';
+        }
+      }
+      return this.chatMetadata.channel;
+    },
```

Comment:
This code is repeated below. @iamsivin Can you please move it to mixin or a util?